src/main/java/com/puppycrawl/tools/checkstyle/checks/sizes/ParameterNumberCheck.java

```diff
@@ -32,6 +34,12 @@
  * </p>
  * <ul>
  * <li>
+ * Property {@code ignoreAnnotatedBy} - Ignore methods and constructors
+ * annotated with the specified annotation(s).
```

Comment:
> Ignore ... constructors annotated with the specified annotation(s).

Where is the test that shows this?

Reply:
I believe you have missed:
https://github.com/checkstyle/checkstyle/pull/14553/files#diff-22ceb90d0660b452426c9a948115933dd3e1afd09a4473174d6179ced132df60
and
https://github.com/checkstyle/checkstyle/pull/14553/files#diff-297d5bdb06dede8c554b1542df9781b8833099a36eb71ae102368195f8b6dcd8

Are they not enough/missing something?

Reply:
I see them now. I recommend when you submit links, you also include the line numbers so I know exactly where you are saying the cases are. Pointing to the whole file doesn't help, when I can already see the file but not the specific cases.

Reply:
I didn't included the line numbers because everything in those two files are the cases concerning these,
if I pointed to the specific line numbers they wouldn't be too helpful without going through the config at the top,
at that point I would have created a whole file in the comment itself.

Reply:
But if you still believe that including the line number in this case can somehow be helpful,
please do tell me so that I can do it for future cases.

Reply:
I wouldn't have mentioned it if I didn't think it would be helpful. This item is resolved.